WISE-Community/WISE-Client
# feat(Tag): Hide archived tag and prevent user from creating archived tag

## Changes
- Hide archived tag in run card, apply tags list, and manage tags dialog
- Prevent user from creating a tag named archived

## Test
- Make sure archived tag does not show up in
  - run card
  - apply tags list
  - manage tags dialog
- Make sure the user can not create a tag named archived
- Make sure the user can not change a tag to be named archived
